tests/Doctrine/Tests/ORM/Functional/EagerFetchCollectionTest.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Query\QueryException;
+use Doctrine\Tests\OrmFunctionalTestCase;
+use Exception;
+
+use function count;
+
+class EagerFetchCollectionTest extends OrmFunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        try {
+            $this->_schemaTool->createSchema([
+                $this->_em->getClassMetadata(EagerFetchOwner::class),
+                $this->_em->getClassMetadata(EagerFetchChild::class),
+            ]);
+        } catch (Exception $e) {
+        }
```

Comment:
You could leverage this method here: https://github.com/doctrine/orm/blob/0b7fe1862e2594fbc77eec8e4af66c42dabba9cd/tests/Doctrine/Tests/OrmFunctionalTestCase.php#L347-L354